packages/react-ui/src/field/Checkbox.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Label } from "./Label";
+import { box, fonts, spacings } from "../theme";
+
+export const InputCheckbox = ({ label, name, id, size, ...props }) => (
+  <StyledLabel htmlFor={id}>
+    <StyledCheckbox
+      type="checkbox"
+      name={name}
+      id={id}
+      size={size}
+      {...props}
+    />
+    {label}
+    <StyledCheckmark size={size} />
+  </StyledLabel>
+);
+
+InputCheckbox.propTypes = {
+  label: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+  size: PropTypes.string
+};
+
+InputCheckbox.defaultProps = {
+  size: "2rem"
+};
+
+const StyledCheckmark = styled.span`
+  position: absolute;
+  top: 0.6rem;
+  left: 0;
+  width: ${props => props.size};
+  height: ${props => props.size};
+  border-radius: ${spacings.tiny};
+  background-color: transparent;
+  border: ${({ theme }) => box.border(theme.border)};
+  &:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+`;
+
+const StyledCheckbox = styled.input`
+  position: relative;
+  display: inline-block;
+  flex-shrink: 0;
+  width: ${props => props.size};
+  height: ${props => props.size};
+  border-radius: ${spacings.tiny};
+  margin: 0 ${spacings.base} 0 0;
+  padding: 0;
+  box-shadow: none;
+  appearance: none;
+  *::-ms-backdrop,
+  & {
+    visibility: hidden;
+  }
+`;
+
+const StyledLabel = styled(Label)`
+  position: relative;
+  font-weight: normal;
+  font-size: ${fonts.sizes.default};
+  cursor: pointer;
+  -webkit-user-select: none;
```

Comment:
this should be handle by styled component

Reply:
Yes

Reply:
Done